src/Camera.py

```diff
@@ -0,0 +1,41 @@
+"""Holds code for the camera device."""
+import cv2
+from numpy import ndarray
+
+
+class Camera:
+    """Abstraction of the cv2 VideoCapture device."""
+
+    TARGET_RESOLUTION_HEIGHT = 480
+
+    def __init__(self) -> None:
+        print("Initializing camera...")
+        # Get access to the VideoCapture device
+        self.capture = cv2.VideoCapture(0)
+        # Calculate an adjusted resolution
+        baseWidth = self.capture.get(cv2.CAP_PROP_FRAME_WIDTH)
+        baseHeight = self.capture.get(cv2.CAP_PROP_FRAME_HEIGHT)
+
+        print(f"Camera resolution: {baseWidth}x{baseHeight}")
+
+        factor = float(Camera.TARGET_RESOLUTION_HEIGHT / baseHeight)
```

Comment:
Uniformly downsampling the captured image to 480p is a good idea for the live video preview, but this would negatively impact the pupil detection, since it drastically reduces the amount of information the detection model has access to when attempting to make a prediction.

Reply:
It would probabally be better to leave the camera alone, and just uniformly downscale the image when rendering it in the live preview, instead of changing the camera settings to limit the resolution of what it captures.